Source/Controllers/SPAppController.m

```diff
@@ -1639,11 +1644,21 @@ - (IBAction)reloadBundles:(id)sender
 																				 format:NULL
 																				  error:&readError];
 						}
-
+						
 						if(!cmdData || readError) {
-							NSLog(@"“%@” file couldn't be read. (error=%@)", infoPath, readError);
-							NSBeep();
-							continue;
+							SPLog(@"“%@” file couldn't be read. (error=%@)", infoPath, readError.localizedDescription);
+							if(![alreadyBeeped objectForKey:bundle]){
+								NSBeep();
+								[alreadyBeeped setObject:@YES forKey:bundle];
+								
+								// remove the dodgy bundle
+								[self removeBundle:bundle];
```

Comment:
Should this be outside of the if/else beep block so that we call delete regardless of beep status? Probably not a big deal either way